### ISSUE TITLE
WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (.u-url)

### ISSUE THREAD

**Issue by aarongustafson** [aarongustafson/aaron-gustafson.com#60]
#### Issue
`Ensures links have discernible text` ([link-name](https://accessibilityinsights.io/info-examples/web/link-name))
#### Target application
[303 Creative LLC v. Elenis is Incredibly Problematic :: Aaron Gustafson](https://www.aaron-gustafson.com/notebook/303-creative-llc-v-elenis-is-incredibly-problematic/)
#### Element path
#webmention-1581314 > .webmention__author.h-card.p-author > .u-url
#### Snippet
`<a href="https://urbanists.social/@curio_research" class="u-url">`
#### How to fix
    Fix all of the following:
      Element is in tab order and does not have accessible text
    
    Fix any of the following:
      Element does not have text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
#### Environment
Microsoft Edge version 112.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
